ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -59,20 +77,30 @@
   private final ClusterTerminator clusterTerminator;
   private final ServerState serverState;
 
+  private final List<Sensor> sensors;
+  private final Map<String, String> customMetricsTags;
+  private final Metrics metrics;
+  private final String ksqlServiceId;
```

Comment:
why do we need to store this as a member?

Reply:
It's used when configuring the metric.

Reply:
Right, but why do we have to store it? Can't we just pass it down?

Reply:
This isn't stored anymore as it's passed to the CommandRunnerStatusMetric class now.